### PR TITLE
Docs for Oas 3.0.0 + Some more fixes

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

**Comment by ForsakenHarmony** [comment]
I have no idea why this is so messed up

might want to cherry pick if possible

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "start": "node ./lib/bin/converter.js",
     "build": "babel src -d lib && flow-copy-source -v src lib",
     "build-and-publish": "rm -rf lib && npm install && npm test && npm run build && npm publish",
-    "eslint": "./node_modules/.bin/eslint ./src index.js ./src",
+    "eslint": "eslint ./src ./test",
```
Comment (jstoiko) [inline package.json:16]: maybe blacklist instead?

File: test/data/raml10-oas30/target/baseUri.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 servers:
-  - url: 'http://93.144.28.153:8081/api/'
+  - url: '93.144.28.153:8081/api/'
```
Comment (jstoiko) [inline test/data/raml10-oas30/target/baseUri.yaml]: aren't URLs supposed to have a scheme here? see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#fixed-fields-4

File: package.json (hunk 1 above)
Comment (ForsakenHarmony) [reply package.json:16]: would require another file `.eslintignore` idk

**Review by tomasmateus — APPROVED** [review]

**Pull request merged**